modules/user_accounts/php/edit_user.class.inc

```diff
@@ -1218,6 +1224,71 @@ class Edit_User extends \NDB_Form
         return $permission;
     }
 
+    /**
+     * This function returns a boolean representing whether or not
+     * to display examiner fields to a user that is editing an account.
+     * Examiner fields SHOULD be displayed in the following cases:
+     *      (1) Editor has 'examiner_multisite' permission
+     *          (which allows to add / certify examiners across all sites).
+     *      (2) Editor has 'examiner_view' permission
+     *          (which allows them to add / certify examiner at their own site)
+     *              AND  either:
+     *                  i) The user is from same site as editor,
+     *                      OR
+     *                  ii) The editor is creating a new user.
+     *
+     * @param \User $editor User editing the form
+     *
+     * @return bool  - whether or not to display examiner fields
+     */
+    function showExaminerFields($editor)
```

Comment:
I find this method name confusing because it sounds like an action, not a question. Maybe something like `shouldShowExaminerFields` would be better (though I'm not entirely happy with that either..)

Also, since this is the major branch can you make sure you add the return type for PHP instead of just the comment?

Reply:
I renamed the function to something a little clearer.

For the return type, none of the other functions in the class have return type indicated. Was i supposed base off of master rather than 21.0-release?